install/0000_90_machine-config-operator_01_prometheus-rules.yaml

```diff
@@ -77,7 +77,8 @@ spec:
       rules:
         - alert: MCDPivotError
           expr: |
-            max by(namespace, node, pod) (increase(mcd_pivot_errors_total[15m])) > 0
+            mcd_pivot_errors_total > 0
+          for: 2m
```

Comment:
I feel that we may revisit this in future to wait a bit longer. But for now, this is fine as it is